### ISSUE TITLE
Notifications Bug

### ISSUE THREAD

**Issue by lshanker** [Purdue-ACM-SIGAPP/BoilerFaves#52]
Someone told me notifications stopped sending after they'd been using the app for a while. Could be an issue with the boot receiver or alarms?

**Comment by yatharthrawat** [comment]
I think this has to do with the fact that we never kill our service and after a point of time the OS does it. 
I could be wrong.

**Comment by lshanker** [comment]
Yeah it's a good thought but I don't think that's it. If you look at the JobService docs it says the service will end if you return false in onStartJob:

> Return true from this method if your job needs to continue running. If you do this, the job remains active until you call jobFinished(JobParameters, boolean) to tell the system that it has completed its work, or until the job's required constraints are no longer satisfied.
> Returning false from this method means your job is already finished. The system's wakelock for the job will be released, and onStopJob(JobParameters) will not be invoked.

https://developer.android.com/reference/android/app/job/JobService
